arm-software/embedded/README.md

```diff
@@ -43,6 +44,10 @@ uses all the latest libc++ improvements and bugfixes, but may result in link
 errors when linking against objects compiled against older versions of the ABI.
 For more information see https://libcxx.llvm.org/DesignDocs/ABIVersioning.html.
 
+Libc++ testing has been fully completed for only a few of the variants. The specific
+variants for which testing is enabled can be found in the corresponding JSON files
+located at the [path](arm-multilib/json/variantis).
```

Comment:
Possible typo here: "variantis". Please fix and also check in the final rendering if the link works.